### DISCUSSION TITLE
Using <C-k> as <Esc>

### DISCUSSION THREAD

**Discussion by RedDocMD** [JetBrains/ideavim#325] [Q&A]
Hi, I am trying to use the following keybindings:
```
nnoremap <C-k> <Esc>
inoremap <C-k> <Esc>
vnoremap <C-k> <Esc>
snoremap <C-k> <Esc>
xnoremap <C-k> <Esc>
cnoremap <C-k> <C-c>
onoremap <C-k> <Esc>
lnoremap <C-k> <Esc>
tnoremap <C-k> <Esc>
```
So basically, I am trying to rebind `Ctrl + K` to `<Esc>`.
**I have set Ctrl+K to VIM (and not IDE or undefined)**
![image](https://user-images.githubusercontent.com/54888685/120066374-a5d72d00-c093-11eb-8eac-d177aacea0f4.png)
For good measure, I have also added the following line in the `.ideavimrc`: `sethandler <C-K> a:vim`.

But it doesn't work. I get the following message in the status bar:
![image](https://user-images.githubusercontent.com/54888685/120066422-e6cf4180-c093-11eb-8057-70c2402dcb7e.png)
And thereafter, it bookmarks the line.

How do I get around this?

**Comment by AlexPl292** [comment]
Hi! It looks like you have some keys in your IJ keymap that start with Ctrl-K. I'm afraid at the moment IdeaVim can't work with this and you have to remove such shortcuts. You can find them on the keymaps menu.
![2021-05-31 09 27 53](https://user-images.githubusercontent.com/4203721/120149607-a3d7b000-c1f2-11eb-8b8e-05ee32e3a160.gif)
